ams/api/src/main/java/com/netease/arctic/ams/api/events/Event.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.ams.api.events;
+
+/**
+ * Standard event interface in Amoro domain this is a parallel concept with specific metrics data of
+ * formats like MetricReport in Apache Iceberg Implementations could include all kinds of process
+ * metrics and resource metrics.
+ */
+public interface Event {
+
+  /** @return event type */
+  EventType type();
+
+  /**
+   * The timestamp of event when it is happened.
+   *
+   * @return event created time.
+   */
+  long eventTime();
```

Comment:
Is `timestampMillis` a proper name?
We should highlight the return value rather than `event`.

Reply:
fixed.